Client/Telemetry/UnifiedTelemetry.swift

```diff
@@ -21,6 +21,16 @@ class UnifiedTelemetry {
         telemetryConfig.isCollectionEnabled = sendUsageData
         telemetryConfig.isUploadEnabled = sendUsageData
 
+        telemetryConfig.measureUserDefaultsSetting(forKey: "profile.blockPopups", withDefaultValue: true)
+        telemetryConfig.measureUserDefaultsSetting(forKey: "profile.saveLogins", withDefaultValue: true)
+        telemetryConfig.measureUserDefaultsSetting(forKey: "profile.showClipboardBar", withDefaultValue: false)
+        telemetryConfig.measureUserDefaultsSetting(forKey: "profile.settings.closePrivateTabs", withDefaultValue: false)
+        telemetryConfig.measureUserDefaultsSetting(forKey: "profile.ASPocketStoriesVisible", withDefaultValue: true)
+        telemetryConfig.measureUserDefaultsSetting(forKey: "profile.ASBookmarkHighlightsVisible", withDefaultValue: true)
+        telemetryConfig.measureUserDefaultsSetting(forKey: "profile.ASRecentHighlightsVisible", withDefaultValue: true)
+        telemetryConfig.measureUserDefaultsSetting(forKey: "profile.prefkey.trackingprotection.enabled", withDefaultValue: "onInPrivateBrowsing")
+        telemetryConfig.measureUserDefaultsSetting(forKey: "profile.prefkey.trackingprotection.strength", withDefaultValue: "basic")
```

Comment:
Constants should be used where possible:
`forKey: ContentBlocker.PrefKeyEnabledState`, withDefaultValue: ContentBlocker.EnabledState.onInPrivateBrowsing`

`forKey: ContentBlocker.PrefKeyStrength`, withDefaultValue: ContentBlockerBlockingStrength.basic`

Reply:
I couldn't do this here easily without a bunch of conditional code since `ContentBlocker` is not defined <iOS 11.